rekcurd/core/rekcurd_worker.py

```diff
@@ -0,0 +1,159 @@
+            result.option: Optional metrics. dict[str, float]
+        :return detail[]: Detail result of each prediction. List[EvaluateResultDetail]
+            detail[].result: Prediction result. PredictResult
+            detail[].is_correct: Prediction result is correct or not. bool
+        """
+        raise NotImplemented()
+
+    @abstractmethod
+    def get_evaluate_detail(self, filepath: str, details: List[EvaluateResultDetail]) -> Generator[EvaluateDetail, None, None]:
+        """
+        get_evaluate_detail
+        :param filepath: Evaluation data file path. str
+        :param details: Detail result of each prediction. List[EvaluateResultDetail]
+        :return rtn: Return results. Generator[EvaluateDetail, None, None]
+            rtn.input: Input data. PredictInput, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.label: Predict label. PredictLabel, one of string/bytes/arr[int]/arr[float]/arr[string]
+            rtn.result: Predict detail. EvaluateResultDetail
+        """
+        raise NotImplemented()
+
+    @property
+    def system_logger(self):
+        return self._system_logger
+
+    @system_logger.setter
+    def system_logger(self, system_logger: SystemLoggerInterface):
+        if isinstance(system_logger, SystemLoggerInterface):
+            self._system_logger = system_logger
+        else:
+            raise TypeError("Invalid system_logger type.")
+
+    @property
+    def service_logger(self):
+        return self._service_logger
+
+    @service_logger.setter
+    def service_logger(self, service_logger: ServiceLoggerInterface):
+        if isinstance(service_logger, ServiceLoggerInterface):
+            self._service_logger = service_logger
+        else:
+            raise TypeError("Invalid service_logger type.")
+
+    def load_config_file(self, config_file: str):
+        self.config = RekcurdConfig(config_file)
+
+    def run(self, host: str = None, port: int = None, max_workers: int = None, **options):
+        import grpc
+        import os
+        import time
+        from concurrent import futures
+        from rekcurd.protobuf import rekcurd_pb2_grpc
+        from rekcurd import RekcurdDashboardServicer, RekcurdWorkerServicer
+
+        if self.config is None:
+            self.config = RekcurdConfig()
+        if port and "service_port" in options:
+            options["service_port"] = port
+        self.config.set_configurations(**options)
+
+        self.data_server = DataServer(self.config)
+        if self._system_logger is None:
+            self._system_logger = JsonSystemLogger(config=self.config)
+        if self._service_logger is None:
+            self._service_logger = JsonServiceLogger(config=self.config)
+        _host = "127.0.0.1"
+        _port = 5000
+        _max_workers = 1
+        host = host or _host
+        port = int(port or self.config.SERVICE_PORT or _port)
+        max_workers = int(max_workers or _max_workers)
```

Comment:
I referred it from Flask implementation. And I believe this can prioritize the user intent.
https://github.com/pallets/flask/blob/master/flask/app.py#L844